fhir-server-test/src/test/java/testng.xml

```
@@ -1,43 +1,53 @@
-            <class name="com.ibm.fhir.server.test.DeleteTest" />
-            <class name="com.ibm.fhir.server.test.UpdateTest" />
-            <class name="com.ibm.fhir.server.test.DuplicateResourceIdTest"/>
-            <class name="com.ibm.fhir.server.test.FHIRDocumentOperationTest" />
-            <class name="com.ibm.fhir.server.test.FHIRValidateOperationTest" />
-            <class name="com.ibm.fhir.server.test.FHIRHealthcheckOperationTest" />
-            <class name="com.ibm.fhir.server.test.PrettyServerFormatTest" />
-            <class name="com.ibm.fhir.server.test.MultiDataStoreTest" />
-            <class name="com.ibm.fhir.server.test.SearchTest" />
-            <class name="com.ibm.fhir.server.test.SearchAllTest" />
-            <class name="com.ibm.fhir.server.test.SearchExtensionsTest" />
-            <class name="com.ibm.fhir.server.test.SortingTest" />
-            <class name="com.ibm.fhir.server.test.SearchNearTest" />
-            
-            <class name="com.ibm.fhir.client.test.FHIRClientTest" />
-
-            <class name="com.ibm.fhir.cli.test.FHIRCliTest" />
-            <!-- requires $hello operation to be deployed. 
-            Include the fhir-operation-[version]-tests jar in the fhir-server userlib dir before running the test:
-            -->
-            <class name="com.ibm.fhir.server.test.FHIROperationTest" />
-            
-            <!-- The following class is intentionally moved to last.  
-                During debug, the other tests interfere with the asynch nature of the test -->
-            <class name="com.ibm.fhir.server.test.WebSocketNotificationsTest" />
-        </classes>
-    </test>
-</suite>
+        <packages>
+            <!-- All of these tests are automatically executed -->
+            <package name="com.ibm.fhir.server.test" />
+        </packages>
+    </test>
+    <test name="ClientTests">
+        <packages>
+            <!-- All of these tests are automatically executed -->
```

Given that `test.client.oauth.enabled = false` by default, I think this comment is misleading

---

They are automatically executed, let me add to the existing statement and clarify.

---

Clarified